flake.nix

```diff
@@ -93,9 +93,9 @@
             ({pkgs, ...}: {
               packages.cardano-cli.configureFlags = [ "--ghc-option=-Werror" ] ++ gitRevFlag;
               packages.cardano-cli.components.tests.cardano-cli-test.build-tools =
-                with pkgs.buildPackages; [ jq coreutils shellcheck ];
+                with pkgs.buildPackages; [ jq coreutils ];
```

Comment:
why do we need `jq` for cli tests? 🤔 I have also doubts about coreutils there.

Reply:
I'll look into this next :+1: